kubernetes/base/leaderelection/electionconfig.py

```diff
@@ -19,7 +19,15 @@
 
 class Config:
     # Validate config, exit if an error is detected
-    def __init__(self, lock, lease_duration, renew_deadline, retry_period, onstarted_leading, onstopped_leading):
+    def __init__(
+            self,
+            lock,
+            lease_duration,
+            renew_deadline,
+            retry_period,
+            onstarted_leading,
+            onstopped_leading,
+            release_on_cancel=False):
```

Comment:
please document the [caveat](https://github.com/kubernetes/kubernetes/blob/10066243df4313e28704d3770742f161ceea0112/staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go#L149-L153)

nit: also "cancel" may not be the most accurate word here, since it was from the Golang context idea.